.github/workflows/package.yml

```diff
@@ -0,0 +1,23 @@
+name: Package Addon
+
+on:
+  push:
+    branches: [ main ]
+    tags: [ "*" ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    env:
+      CF_API_KEY: ${{ secrets.CF_API_KEY }}
```

Comment:
@Gogo1951 please create a [CurseForge API key](https://authors.curseforge.com/account/api-tokens) then upload as a `CF_API_KEY` [GitHub Secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) to this repo

Reply:
Via DMs, this was added.